doc/rst/platforms/cray.rst

```diff
@@ -134,8 +130,8 @@ built from source on Cray CS systems using the
 Building Chapel for a Cray System from Source
 ---------------------------------------------
 
-1) If using a XC, XE, or XK system, continue to step 2. If using a
-   CS series system, set ``CHPL_HOST_PLATFORM`` to ``cray-cs``.
+1) If using a XC system, continue to step 2. If using a CS series
```

Comment:
Think this should be (should always have been) "an" XC system, because of the initial vowel phoneme.